src/views/wedo2/wedo2.jsx

```diff
@@ -44,6 +44,14 @@ var Wedo2 = React.createClass({
                                 </h4>
                                 <p>
                                     <FormattedHTMLMessage id='wedo2.installText' />
+                                    <br />
+                                    <a href="https://itunes.apple.com/WebObjects/MZStore.woa/wa/viewSoftware?id=1084869222&mt=12">
```

Comment:
@rschamp looks like the linter is failing on this again – I assume I should get a bitly link for now, since we still need to add in the global linting rules for www?